akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -53,6 +51,10 @@ private[akka] abstract class SinkModule[-In, Mat](val shape: SinkShape[In]) exte
     if ((thatN eq null) || thisN == thatN) shape
     else shape.copy(in = Inlet(thatN + ".in"))
   }
+
+  protected def label: String
+  final override def toString: String = f"$label [${System.identityHashCode(this)}%08x]"
```

Comment:
@bantonsson doesn't identityHashCode cause inflation of the object header to store the current address of the pointer to the object?